### PR TITLE
BG 33901: Moved block exploring into coin controllers

### PR DESCRIPTION
**Pull request by raovinnakota**
Card: BG 33901: https://bitgoinc.atlassian.net/browse/BG-33091?filter=-1
Abstracting the tx explorer from recovery to UTXO coin controller

### CODE REVIEW

**Comment by raovinnakota** [comment]
Moved the transaction explorer into the coin controller so that specific coins can override with 3rd party explorers when ready to implement

File: modules/core/src/v2/coins/abstractUtxoCoin.ts [hunk 1]
```diff
@@ -1700,6 +1700,14 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
   protected abstract getAddressInfoFromExplorer(address: string, apiKey?: string): Bluebird<AddressInfo>;
   protected abstract getUnspentInfoFromExplorer(address: string, apiKey?: string): Bluebird<UnspentInfo[]>;
 
+  getUnspentFromWrongChain(faultyTxId: string): any {
```
Comment (tylerlevine) [inline modules/core/src/v2/coins/abstractUtxoCoin.ts:1703]: Name here seems a bit strange - maybe something like `getTxInfoFromExplorer` would be a bit more descriptive? As it is, this doesn't have much to do with unspents or wrong chains.

**Review by tylerlevine — CHANGES_REQUESTED** [review]
comment on method name, but other than that lgtm

File: modules/core/src/v2/coins/abstractUtxoCoin.ts (hunk 1 above)
Comment (raovinnakota) [reply modules/core/src/v2/coins/abstractUtxoCoin.ts:1703]: Yeah that sounds good. The tests used to the "Wrong Chain" as the descriptor, which led me to name it that. Going to change now.

Comment (raovinnakota) [reply modules/core/src/v2/coins/abstractUtxoCoin.ts]: Change pushed @tylerlevine

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**